### PR TITLE
Silver swords

### PR DESCRIPTION
**Pull request by Marmio64**
### Intent of your Pull Request

to add silver swords
They have the same model as the claymore since claymores already look kind of silverish.
### Why is this change good for the game?

lets you start anti-vampire crusades

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.

adds silver swords. They deal only 12 damage by default, but can reveal vampires and deal extra burn damage to them. They are only purchasable in cargo and require sec access to open, and they also cost 5,000 credits for 3 of them (laser guns are 2,000 for 3)

### What should players be aware of when it comes to the changes your PR is implementing?

^^^

### What general grouping does this PR fall under? 
For example, Engineering changes, Medical rebalancing, TEG tweaks, etc.?

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
no
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
adds silver swords. They deal only 12 damage by default, but can reveal vampires and deal extra burn damage to them. They are only purchasable in cargo and require sec access to open, and they also cost 5,000 credits for 3 of them (laser guns are 2,000 for 3)

# Changelog

:cl:  
rscadd: Adds silver swords
/:cl:

### CODE REVIEW

**Comment by ChesterTheCheesy** [comment]
validhunting tool hooray

**Comment by Jarod1200** [comment]
Yeah I have to say. Isn't this literally a validhunt stick

**Comment by Marmio64** [comment]
> Yeah I have to say. Isn't this literally a validhunt stick

basically, but its more meant to be something security orders for themselves or to arm deputies with against the vampire horde, since it requires sec access to open.

**Comment by ChesterTheCheesy** [comment]
Sec access is a joke, a paranoid chaplain can order 5 of those, ask QM to dump the crates in lava to open them, and make a small validhunt army.

Maybe lock it behind the crate behind hacking the console, and make the crate only hold 2 swords.

**Comment by Marmio64** [comment]
> Sec access is a joke, a paranoid chaplain can order 5 of those, ask QM to dump the crates in lava to open them, and make a small validhunt army.
> 
> Maybe lock it behind the crate behind hacking the console, and make the crate only hold 2 swords.

Sec will have to do their job and protect innocent vamps, since the eotc law was removed

**Comment by UselessTheremin** [comment]
i mean in what way does this make the game better, like the flaws seem to really outweigh the positive. is the positive just "having more stuff?" because bloat is definitely not a good thing, especially with shit like this that is literally only used for validhunting. if anything there needs to be more non-sec crates as that section is bloated to high hell and is incredibly powerful. what i want is there to be more balance to crates and not this tbh

**Comment by ChesterTheCheesy** [comment]
> 
> 
> > Sec access is a joke, a paranoid chaplain can order 5 of those, ask QM to dump the crates in lava to open them, and make a small validhunt army.
> > Maybe lock it behind the crate behind hacking the console, and make the crate only hold 2 swords.
> 
> Sec will have to do their job and protect innocent vamps, since the eotc law was removed

"Vampires and lings go under the murder crime based on how their mechanics work."

**Comment by Marmio64** [comment]
> validhunting tool hooray



> > > Sec access is a joke, a paranoid chaplain can order 5 of those, ask QM to dump the crates in lava to open them, and make a small validhunt army.
> > > Maybe lock it behind the crate behind hacking the console, and make the crate only hold 2 swords.
> > 
> > 
> > Sec will have to do their job and protect innocent vamps, since the eotc law was removed
> 
> "Vampires and lings go under the murder crime based on how their mechanics work."

lings are automatically murderers, but vampires are innocent by default and can drink blood from willing participants, which makes it legal.

**Comment by Marmio64** [comment]
but, since the general consensus is no, i'll go ahead and close this
